src/strategies/aavegotchi-agip/index.ts

```diff
@@ -85,9 +88,26 @@ export async function strategy(
     if (itemValue > 0) prices[parseInt(itemInfo.svgId)] = itemValue;
   });
 
+  // agip 17: Voting power of 0.5 GHST/pixel
+  const realmSizeVotePower = {
+    0: 32, // humble
+    1: 128, // reasonable
+    2: 1028, // spacious vertical
+    3: 1028, // spacious horizontal
+    4: 2048 // partner
+  };
+
   const walletScores = {};
-  result.users.map((addrInfo) => {
-    const { id, gotchisOwned } = addrInfo;
+  result.users.map(({ id, gotchisOwned, parcelsOwned }) => {
+    let realmVotingPowerValue = 0;
+    if (parcelsOwned.length > 0) {
+      parcelsOwned.map(({ size }) => {
+        let votePower = realmSizeVotePower[size];
+        if (isNaN(votePower)) votePower = 0;
+        realmVotingPowerValue += votePower;
+      });
+    }
```

Comment:
Hi @candoizo Existing proposals could be affected by this change, especially the proposal that is open 
https://snapshot.org/#/aavegotchi.eth/proposal/0x396844d504394f7f335f070d443c33455300ee21d90db6c4b089760a3e0469bf

It is better to create a `-v2` strategy for this

Reply:
alright ive made the changes requested